examples/blender/blender.py

```diff
@@ -1,11 +1,15 @@
+#!/usr/bin/env python3
+
 from yapapi import enable_default_logger
 from yapapi.runner import Engine, Task, vm
 from yapapi.runner.ctx import WorkContext
 from datetime import timedelta
+import argparse
 import asyncio
+import logging
 
 
-async def main():
+async def main(subnet_tag="testnet"):
```

Comment:
I would remove `main` argument from here and add a call to `build_parser` below.

Reply:
(Unless you want to call main() directly.)

Reply:
I prefer main() to stay parametrized (as it can be called directly)